### DISCUSSION TITLE
Github desktop "Your account token has been invalidated..."

### DISCUSSION THREAD

**Discussion by heidarj** [community/community#29594] [Other Feature Feedback, Questions, & Ideas] [answered]
Suddenly my GIthub desktop client has started showing a pop-up every 15 minutes and asking me to sign in again. The message I get is "Your account token has been invalidated and you have been signed out from your Github account. Do you want to sign in again?"
I tried signing out of the client and in again, but still same issue. This is version 3.0.5

**Comment by shiftkey** [comment, marked as answer]
:wave: I'd recommend reaching out to the issue tracker for GitHub Desktop as they'll be able to help investigate your issue https://github.com/desktop/desktop/issues

**Reply by heidarj** [reply]
Thank you, I was looking for the right place to post this, thought this was it :)
